doc/sphinx/source/esmvaldiag/observations.rst

```diff
@@ -0,0 +1,283 @@
+************************************************************
+Contributing a CMORizing script for an observational dataset
+************************************************************
+
+ESMValTool is able to process observational data sets, however, since many
```

Comment:
This paragraph is confusing CF and CMOR. 
I would simply write that the ESMValTool is designed to work with CF compliant data and follows the CMOR tables from the CMIP data request, therefore the observational datasets need to be CMORized for usage in ESMValTool.

Reply:
I changed the wording.